pkg/common/utils.go

```diff
@@ -251,15 +255,39 @@ func ValidateSnapshotType(snapshotType string) error {
 	}
 }
 
-// ConvertGiBStringToInt64 converts a GiB string to int64
-func ConvertGiBStringToInt64(str string) (int64, error) {
+// ConvertStringToInt64 converts a string to int64
+func ConvertStringToInt64(str string) (int64, error) {
 	// Verify regex before
-	match, _ := regexp.MatchString("^([+-]?[0-9.]+)([eEinumkKMGTP]*[-+]?[0-9]*)$", str)
+	match, _ := regexp.MatchString("^([+-]?[0-9.]+)([mkKM]*[-+]?[0-9]*)$", str)
 	if !match {
 		return 0, fmt.Errorf("invalid string %s", str)
 	}
-	quantity := resource.MustParse(str)
-	return volumehelpers.RoundUpToGiB(quantity)
+
+	multiplier := int64(1)
+	value, err := strconv.ParseInt(str, 10, 64)
+	if err != nil {
+		// the string ends with m, M or k, K
+		last := string(str[len(str)-1])
+
+		switch last {
+		case "k":
+			multiplier = thousand
+		case "K":
+			multiplier = thousand
+		case "m":
```

Comment:
"m" means milli, not mega

Reply:
Here we do million and thousand?

Reply:
In the SI system, the exact letters and capitalization is important.
M = 10^6
m = 1/1000